python/example_code/cloudfront/list_distributions.py

```diff
@@ -12,21 +12,24 @@
 # language governing permissions and limitations under the License.
 
 import boto3
-
+from __future__ import print_function
```

Comment:
you need to place this line before you import boto3

Reply:
Thanks, good catch.